tests/unit/conftest.py

```diff
@@ -5,6 +5,5 @@
 
```

Comment:
typo in filename. `conftest` is a special file in pytest. Read: https://www.testinglpoint.com/pytest-conftest-py/